### PR TITLE
Proposal for a restructuring of this repository

### PR DESCRIPTION
**Pull request by prrrnd**
I would like to propose a restructuring of this repository.

- axp
  - README.md 
In this file, we would explain what AXP stands for and what it means for Abtioneers, as well as practical info for using `git pair`, different keyboard layouts, etc.
- career-development
  - ladder.md
  - roles.md
No changes here, besides renaming of the folder and perhaps merging of the files into a README.md later down the road.
- javascript
No changes
- rails
I would like to extract the template.rb to its own repository, and redirect most of the information to abtion/muffi.
- services
  - README.md
Listing the services we prefer to use.
- wordpress
.pairs
README.md
README.standard.md 

Ideally, we would extract the standard README to its own repo, so it's in one place and easy to maintain. Right now, there is confusion between abtion/muffi and this repository.

### CODE REVIEW

File: .pairs [hunk 1]
```diff
@@ -7,6 +7,7 @@ pairs:
   ln: Lasse Norfeldt; ln
   gh: Geoff Hubbard; gh
   tbb: Thor-Bjørn Böhme; tb
+  pb: Pierre-Arnaud Baciocchini; pb
```
Comment (GeoffAbtion) [inline .pairs]: Unrelated? `.pairs` edit

**Comment by GeoffAbtion** [comment]
Is it worthwhile documenting this structure in a markdown file also?

File: .pairs (hunk 1 above)
Comment (runephilosof-abtion) [reply .pairs]: Agreed. Otherwise good.

Comment (heatherm) [reply .pairs]: Fine with this. Also happy to remove and use github's co-authoring https://help.github.com/en/github/committing-changes-to-your-project/creating-a-commit-with-multiple-authors

**Comment by heatherm** [comment]
Nice all, looks great!

**Pull request merged**